main.rb

```diff
@@ -0,0 +1,175 @@
+# rubocop:disable Style/CyclomaticComplexity
+require './book'
+require './person'
+require './teacher'
+require './student'
+require './rental'
+
+class Library
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      print 'No books in library'
+      return
+    end
+    @books.each do |book|
+      print "Title: #{book.title.capitalize}, Author: #{book.author.capitalize}\n"
+    end
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp.capitalize
+    print 'Author: '
+    author = gets.chomp.capitalize
+    book = Book.new(title: title, author: author)
+    @books << book
+    puts "Book created successfully\n"
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'Your Library is empty'
+      return
+    end
+    @people.each do |person|
+      print "[#{person.class.name}] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age}\n"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
```

Comment:
- [ ] I am getting an error  `wrong number of arguments (given 0, expected 2..3) (ArgumentError)` when trying to create a student and a teacher: 
<img width="1195" alt="Screenshot 2022-06-16 at 12 36 13" src="https://user-images.githubusercontent.com/77038610/174029230-b7167c97-b363-4094-a9ce-950df09f1d6d.png">